imagedephi/main.py

```diff
@@ -24,12 +27,31 @@ def imagedephi() -> None:
     "input-dir", type=click.Path(exists=True, file_okay=False, readable=True, path_type=Path)
 )
 @click.argument(
-    "output-dir", type=click.Path(exists=True, file_okay=False, writable=True, path_type=Path)
+    "output-dir", type=click.Path(exists=True, file_okay=False, readable=True, path_type=Path)
 )
-def run(input_dir: Path, output_dir: Path) -> None:
-    """Run in CLI-only mode."""
-    redact_images(input_dir, output_dir)
-    click.echo("Done!")
+@click.argument("override-rules", type=click.File("r"), required=False)
+def run(input_dir: Path, output_dir: Path, override_rules: Optional[TextIO]):
+    """Redact images in a folder according to given rule sets."""
+    base_rules_bytes = pkgutil.get_data("imagedephi", "base_rules.yaml")
```

Comment:
The better-supported way to do this is to use [`importlib.resources.files`](https://docs.python.org/3/library/importlib.resources.html#importlib.resources.files), which supports implicit namespace packages (which means you don't have to have an `__init__.py` in a resource folder for it to be loadable).

```suggestion
    base_rules_path = importlib.resources.files("imagedephi") / "base_rules.yaml"
```

In this case, `base_rules_path` is a [`Traversable`](https://docs.python.org/3.10/library/importlib.html#importlib.abc.Traversable), which supports basically a subset of the `pathlib.Path` API. So, you can:
```python
with base_rules_path.open() as base_rules_stream:
    yaml.safe_load(base_rules_stream)
```

Reply:
Thanks for the pointer. Implemented this change in c234aa9